.github/workflows/ci-checks.yml

```diff
@@ -0,0 +1,87 @@
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+name: Standard CI checks
+
+jobs:
+  Hadolint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: hadolint/hadolint-action@v1.5.0
+        with:
+          dockerfile: Dockerfile
+
+  Code_security:
```

Comment:
noice, https://github.com/district0x/hegex/pull/46/checks?check_run_id=2914527628 pops up 2 critical-lvl vulns, one is from clojure, I think it's not applicable in cljs context in our case; lodash one likely not harmful too since I've seen it on several projects but I'll try to fix them in deps this weeknd